client/memez/projects/memez/src/app/stores/entities/post.store.ts

```diff
@@ -1,69 +1,96 @@
-import {Injectable} from '@angular/core';
-import {action, observable} from 'mobx-angular';
-import {RootStore} from '../root.store';
-import {IPost} from '../../types/Entities/IPost';
+import {Injectable}                   from '@angular/core';
+import {action, computed, observable} from 'mobx-angular';
+import {RootStore}                    from '../root.store';
+import {IPost}                        from '../../../../../../../../shared/types/Entities/IPost';
+import {autorun}                      from 'mobx';
 
-import * as dayjs from 'dayjs';
-import {autorun} from 'mobx';
-import {MOCK_POSTS} from '../../../../../../../../shared/mock/MOCK_POSTS';
-import {ILike} from '../../types/Entities/ILike';
 
 @Injectable({
   providedIn: 'root'
 })
 export class PostStore {
   @observable posts: IPost[] = [];
-  useMock: boolean = true;
+
+  /*useMock: boolean = false;*/
 
   constructor(
     public root: RootStore
   ) {
     this.root.ps = this;  //self-registration at root store
     window['postStore'] = this;
+    autorun(() => {
+
+    })
+
+
+  }
+
+  @computed get reversedPosts() {
+    return this.posts
+      .slice()
+      .reverse()
+  }
 
+  @computed get userPosts() {
+    const userPosts: IPost[] = this.posts
+      .filter(post => post.postedBy._id === this.root.log.currentUser._id)
+    return userPosts.reverse()
```

Comment:
wont it be easier to just return the posts from the server sorted by the timestamp? 

instead of having all these computed functions, in the db.controller.getPosts() do this: 

postCollection.find().sort({timestamp: -1})

this will return them sorted from last to first